src/emulator/main.cpp

```diff
@@ -155,6 +159,9 @@ int main(int argc, char *argv[]) {
 
         set_window_title(host);
     }
+#ifdef NDEBUG
+    discord::shutdown();
```

Comment:
I think it's about time for a host_destroy. Just organizes these destroy functions into a nice clean place. I was going to do it in a GDB pr soon anyway. I want to hear what you guys think.

Reply:
not sure if discord and gdb deinit are host things though, they have no relation to the actual `host` library do they? at least not discord idk about gdb

Reply:
disregard, I didn't realize we initialized HostState in `emulator/init.cpp` (for some reason)